src/sentry/tasks/auto_ongoing_issues.py

```diff
@@ -108,7 +111,7 @@ def schedule_auto_transition_issues_new_to_ongoing(
     to be updated in a single run. However, we expect every instantiation of this task
     to chip away at the backlog of Groups and eventually update all the eligible groups.
     """
-
+    span = sentry_sdk.Hub.current.scope.span
```

Comment:
i think this is `current_hub = sentry_sdk.Hub(sentry_sdk.Hub.current)`

Reply:
or maybe both https://github.com/getsentry/sentry/blob/master/src/sentry/scim/endpoints/utils.py#L21 this is wack